app/js/Routes.jsx

```diff
@@ -8,18 +8,16 @@
  */
 import React, { Component } from 'react';
 import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
-import App from './components/App';
-import HomePage from './components/homePage/HomePage';
-import ManageApps from './components/homePage/manageApps/ManageApps';
-import ManageSettings from './components/homePage/ManageSettings';
+import App from './components/App.jsx';
+import ManageApps from './components/manageApps/ManageApps.jsx';
+import ManageSettings from './components/manageApps/ManageSettings.jsx';
 
 export default () => {
   return (
     <Router history={hashHistory}>
       <Route path="/" component={App} >
-        <IndexRoute component={HomePage}/>
+        <IndexRoute component={ManageApps}/>
         <Route path="manageSettings" component={ManageSettings} />
-        <Route path="manageApps" component={ManageApps} />
       </Route>
```

Comment:
I think we should have a fallback route for when the user tries to access an incorrect URL, I would recommend using the `ManageApp` page as the fallback route.
Adding `<Route path="*" component ={ManageApps}/>` should solve this.

Reply:
That was not originally in the plan but it sure sounds good.